fe1-web/src/features/rollCall/components/__tests__/EventRollCall.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react-native';
+import STRINGS from 'resources/strings';
+import { Hash, Timestamp } from 'core/objects';
+import { useNavigation } from '@react-navigation/native';
+
+import * as reactRedux from 'react-redux';
+import { mockLao } from '__tests__/utils/TestUtils';
+import { requestOpenRollCall, requestReopenRollCall } from 'features/rollCall/network';
+import EventRollCall from '../EventRollCall';
+import { EventTypeRollCall, RollCall, RollCallStatus } from '../../objects';
```

Comment:
You should write imports in the following order (ideally separated with an empty line) :
1. External libraries
2. Everything outside the current feature: core, resources (and other features if needed, but try to avoid this)
3. Everything inside the feature (in this case contained in `features/rollCall`) that can be imported relatively using `../`